### PR TITLE
JESI-1743

### PR DESCRIPTION
**Pull request by AndreTheHunter**
[JESI-1743](https://jesims.atlassian.net/browse/JESI-1743)
- [x] https://github.com/jesims/parent/pull/46
- [x] https://github.com/jesims/bindle/pull/13
- [x] Update changelog
- [x] https://github.com/jesims/bindle/pull/15

### CODE REVIEW

File: package-dry.json [hunk 1]
```diff
@@ -1,36 +0,0 @@
-{
```
Comment (AndreTheHunter) [inline package-dry.json:1]: Moved into bindle. Deleted this file doesn't add anything not in bindle

File: project.clj [hunk 2]
```diff
@@ -8,9 +8,9 @@
             :distribution :repo
             :comments     "same as Clojure"}
   :plugins [[lein-parent "0.3.7"]]
-  :parent-project {:coords  [io.jesi/parent "3.3.0"]
+  :parent-project {:coords  [io.jesi/parent "3.9.0"]
                    :inherit [:plugins :managed-dependencies :deploy-repositories :dependencies :profiles :test-refresh :aliases :codox]}
-  :managed-dependencies [[io.jesi/backpack "4.2.0"]]        ;TODO use manged version once 5.0 is released
+  :managed-dependencies [[io.jesi/backpack "5.2.0"]]
```
Comment (axrs) [inline project.clj:13]: Does backpack need backpack? I think this can be removed now

File: project.clj [hunk 3]
```diff
@@ -23,10 +23,11 @@
...
-                 [org.clojure/clojurescript :scope "provided"]
+                 [thheller/shadow-cljs :scope "provided"]
                  [org.clojars.mmb90/cljs-cache "0.1.4"]     ;TODO move to separate cache project
                  [com.cognitect/transit-cljs "0.8.256"]     ;TODO move to separate http project
                  [com.lucasbradstreet/cljs-uuid-utils "1.0.2"]]
-  :profiles {:dev [:parent/dev {:dependencies [[io.jesi/customs "1.0.0-SNAPSHOT"] ;FIXME use managed version
+  :profiles {:dev [:parent/dev {:dependencies [[io.jesi/customs]
                                                [org.clojure/tools.namespace "0.3.1"]]}]}
-  :clean-targets ^{:protect false} ["target" ".shadow-cljs"])
+  :clean-targets ^{:protect false} [".shadow-cljs" ".cljs_node_repl" "out" :target-path]
```
Comment (axrs) [inline project.clj:32]: Worth killing .clj-kondo?

File: src/io/jesi/backpack/env.cljc [hunk 4]
```diff
@@ -14,30 +14,29 @@
   (boolean (:ns env)))
 
 ;TODO better runtime detection that doesn't require predicates
+;TODO is "runtimes" the correct term? https://clojure.org/guides/reader_conditionals calls them "platforms"
```
Comment (axrs) [inline src/io/jesi/backpack/env.cljc:17]: I think platforms is a better term

File: src/io/jesi/backpack/env.cljc [hunk 5]
```diff
@@ -79,22 +84,33 @@
...
+  to be transformed.
+
+  WARNING: Will throw StackOverflowError for deeply nested forms
+  Use `io.jesi.backpack.env/symbol` to transform individual symbols"
   [env quoted-form]
   {:pre [(not (symbol? quoted-form))]}
-  (let [converter (converter (runtime env))]
-    (if (identical? identity converter)
+  (let [converter (converter env)]
+    (if (nil? converter)
```
Comment (axrs) [inline src/io/jesi/backpack/env.cljc:101]:
```suggestion
  (if-let [converter (converter env)]
```

File: project.clj (hunk 3 above)
Comment (AndreTheHunter) [reply project.clj:32]: Nah, the cache seems to be quite good a dealing with changing deps and versions. Don't think it's worth killing it

File: project.clj (hunk 2 above)
Comment (AndreTheHunter) [reply project.clj:13]: Things that backpack uses use backpack e.g. customs

**Pull request merged**